pyproject.toml

```diff
@@ -32,24 +32,24 @@ classifiers = [
     "Topic :: Security",
 ]
 dependencies = [
-    "tqdm==4.66.3",
-    "pyyaml==6.0.1",
-    "tabulate==0.9.0",
-    "colorama==0.4.6",
-    "termcolor==2.4.0",
-    "wcwidth==0.2.13",
-    "ida-settings==2.1.0",
-    "viv-utils[flirt]==0.7.9",
-    "halo==0.0.31",
-    "networkx==3.1",
-    "ruamel.yaml==0.18.6",
-    "vivisect==1.1.1",
-    "pefile==2023.2.7",
-    "pyelftools==0.31",
-    "dnfile==0.14.1",
-    "dncil==1.0.2",
-    "pydantic==2.7.1",
-    "protobuf==5.26.1",
+    "tqdm>=4.66.3",
```

Comment:
would you please add a comment that indicates something like "the following minimum dependency versions were imported in May 2024". 

then if we have to bump some versions up, or otherwise restrict, we can move those to another section so that it's clear which are "hard" lower limits and which happen to be snapshots from today. 

does this make sense? open to discussion.